### PR TITLE
[receiver/purefa] Add missing reloadIntervals

### PR DESCRIPTION
**Pull request by dgoscn**
**Description:**
Apparently, the non-array metrics are not currently working as expected. This PR comes to fix this.

**Link to tracking Issue:**
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/14886
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16992

**Documentation:**
Still in progress

### CODE REVIEW

**Comment by TylerHelmuth** [comment]
pinging @dgoscn @chrroberts-pure as code owners

**Review by TylerHelmuth — COMMENTED** [review]
@dgoscn are you able to add any unit tests that ensure this fixes the issue?

**Review by TylerHelmuth — COMMENTED** [review]
Also please add a changelog entry

**Comment by dgoscn** [comment]
> @dgoscn are you able to add any unit tests that ensure this fixes the issue?

Yes Tyler, I totally missed this. Submitting asap.
Thanks

File: .chloggen/purefa-receiver-missing-scrapers.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receiver/purefareceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Apparently, the non-array metrics are not currently working as expected.
```
Comment (TylerHelmuth) [inline .chloggen/purefa-receiver-missing-scrapers.yaml:8]: The note should reflect the fix that was made.  Something like "Fixed issue where non-array metrics were not working as expected" but try to specify how they were not working.

File: .chloggen/purefa-receiver-missing-scrapers.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receiver/purefareceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: The changes reflect on the solution of non-array scrapers that were not working as expected
```
Comment (jpkrohling) [inline .chloggen/purefa-receiver-missing-scrapers.yaml:8]:
```suggestion
note: Set an explicit reload interval for all scrapers.
```

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**